channeldb/migration12/invoices.go

```diff
@@ -105,12 +112,6 @@ type Invoice struct {
 	// or any other message which fits within the size constraints.
 	Memo []byte
 
-	// Receipt is an optional field dedicated for storing a
-	// cryptographically binding receipt of payment.
-	//
-	// TODO(roasbeef): document scheme.
-	Receipt []byte
-
```

Comment:
Wasn't `Receipt` already gone?

Reply:
not from the invoice, it was still present on the data structure itself since the migration hadn't been performed. prior it was removed from the rpc and everywhere outside of channeldb